src/NuGet.Core/NuGet.Packaging/PackageExtraction/PackagePathHelper.cs

```diff
@@ -65,7 +65,7 @@ internal static IEnumerable<string> GetDirectories(string root, string path)
 
         private static IEnumerable<string> GetPackageFiles(string root, string filter)
         {
-            filter = filter ?? "*" + PackagingCoreConstants.NupkgExtension;
+            filter ??= "*" + PackagingCoreConstants.NupkgExtension;
```

Comment:
This is just syntactic sugar no?
We shouldn't make the change in that case, as it's the only change in this file.